docs/proposals/04-nat-gateway.md

```diff
@@ -69,7 +69,7 @@ status:
 ```
 
 ```yaml
-apiVersion: networking.onmetal.de/v1alpha1
+apiVersion: networking.api.onmetal.de/v1alpha1
 kind: NATGatewayRouting
```

Comment:
Which object should be reconciled **NATGateway** or **NATGatewayRouting** ? 
And Does **NatGatewayRouting** gets created/updated automatically?

Reply:
The `onmetal-api` controllers will apply `NATGatewayRouting` from `NATGateway`.

Reply:
yes identical to LoadBalancerRouting.